tests/unit/sagemaker/test_serializers.py

```diff
@@ -19,9 +19,14 @@
 import numpy as np
 import pytest
 
-from sagemaker.serializers import CSVSerializer, NumpySerializer, JSONSerializer
+from sagemaker.serializers import CSVSerializer, NumpySerializer, JSONSerializer, SparseMatrixSerializer
 from tests.unit import DATA_DIR
 
+try:
+    import scipy
+except ImportError as e:
+    scipy = DeferredError(e)
```

Comment:
I'm unsure if I should use `DeferredError` here.

Reply:
no need to do so in the tests